packages/esm-service-queues-app/src/send-patient-back-toqueue/send-back-patient-toqueue.component.tsx

```diff
@@ -0,0 +1,288 @@
+import React, { useMemo } from 'react';
+import {
+  Button,
+  ModalBody,
+  ModalFooter,
+  ModalHeader,
+  Form,
+  ContentSwitcher,
+  Switch,
+  Select,
+  SelectItem,
+  InlineNotification,
+  RadioButton,
+  RadioButtonGroup,
+  InlineLoading,
+  Tile,
+  TextInput,
+} from '@carbon/react';
+import { useTranslation } from 'react-i18next';
+import { navigate, showSnackbar, useConfig } from '@openmrs/esm-framework';
+import { useQueueLocations } from '../patient-search/hooks/useQueueLocations';
+import styles from './send-back-patient-toqueue.scss';
+import { useQueues } from '../hooks/useQueues';
+import { useForm, Controller } from 'react-hook-form';
+import { z } from 'zod';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { type ConfigObject } from '../config-schema';
+import { useMutateQueueEntries } from '../hooks/useQueueEntries';
+import { useQueueEntry } from './send-back-patient-toqueue.resource';
+import { updateQueueEntry } from '../active-visits/active-visits-table.resource';
+
+interface SendBackPatientProps {
+  patientUuid: string;
+  closeModal: () => void;
+}
+
+const SendBackPatientToQueue: React.FC<SendBackPatientProps> = ({ closeModal, patientUuid }) => {
+  const { t } = useTranslation();
+  const { concepts } = useConfig<ConfigObject>();
+  const { data, error, isLoading } = useQueueEntry(patientUuid);
+  const allowedPriorities = data?.queue?.allowedPriorities ?? [];
+  const allowedStatuses = data?.queue?.allowedStatuses ?? [];
+  const uppercaseText = (text: string) => text.toUpperCase();
+  const defaultWaitStatus = concepts?.defaultStatusConceptUuid;
+  const defaultResultReviewUrgencyStatus = concepts?.defaultResultReviewPriorityConceptUuid;
```

Comment:
The provider can modify the queue or use the previously loaded queue data to move patients to the previous queue or modify the queue because the default settings for priority, status, and mapping data from the config-schemas.ts and hook are prefill.